hw/ip/kmac/dv/cov/kmac_unmasked_cov_excl.el

```diff
@@ -0,0 +1,121 @@
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_reqfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_sramreqfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_rspfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_reqfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_sramreqfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_rspfifo
+CHECKSUM: "1529365951"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo
+CHECKSUM: "1529365951"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter
+CHECKSUM: "2454292572"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_err
+CHECKSUM: "2454292572"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_err
+CHECKSUM: "3161267887 1427585211"
+INSTANCE: tb.dut.u_sha3.u_keccak
+Fsm keccak_st "1427585211"
+ANNOTATION: "[UNSUPPORTED] unused in unmasked KMAC configuration"
+Transition StPhase3->StPhase1 "4->2"
+Fsm keccak_st "1427585211"
+ANNOTATION: "[UNSUPPORTED] unused in unmasked KMAC configuration"
+Transition StIdle->StPhase1 "0->2"
+Fsm keccak_st "1427585211"
+ANNOTATION: "[UNSUPPORTED] unused in unmasked KMAC configuration"
+Transition StPhase1->StPhase2 "2->3"
+Fsm keccak_st "1427585211"
+ANNOTATION: "[UNSUPPORTED] unused in unmasked KMAC configuration"
+Transition StPhase2->StPhase3 "3->4"
+Fsm keccak_st "1427585211"
+ANNOTATION: "[UNSUPPORTED] unused in unmasked KMAC configuration"
+Transition StPhase3->StIdle "4->0"
+CHECKSUM: "1242460059 1964995507"
+INSTANCE: tb.dut.u_kmac_core
+ANNOTATION: "[UNR] cannot hit due to msgfifo behaviors"
```

Comment:
Are these UNR all because of VCS UNR tool limitations/bugs? if so, would like to know what tool limitations could be (like fifo depth), in order to understand the tool better.

Reply:
tagging @eunchan to confirm - from our discussion yesterday this UNR is due to a small oversight while writing the RTL, perhaps the tool thinks that this condition can be hit

Reply:
Unmasked version has no chance to hit the StPhase3 state. It sits in `StActive` always.